From 8fb6b87467acf779020d47540952c13c8eb9bc07 Mon Sep 17 00:00:00 2001
From: Carlos Soriano <csoriano@redhat.com>
Date: Tue, 11 Dec 2018 19:30:37 +0100
Subject: [PATCH 47/55] desktopGrid: Don't fill grid with placeholder on
 initialization

We might not even have an allocation ready. So delay its initialization
until we have a proper width and height coming from the monitor
constraint.
---
 desktopGrid.js | 2 --
 1 file changed, 2 deletions(-)

diff --git a/desktopGrid.js b/desktopGrid.js
index 2ac89c6..dcf9edf 100644
--- a/desktopGrid.js
+++ b/desktopGrid.js
@@ -116,8 +116,6 @@ var DesktopGrid = class {
         this._grid.connect('key-press-event', this._onKeyPress.bind(this));
 
         this._grid.connect('allocation-changed', () => Extension.desktopManager.scheduleReLayoutChildren());
-
-        this.reset();
     }
 
     _onKeyPress(actor, event) {
-- 
2.20.0

